### PR TITLE
Added script and secure variable for dependent build on-taskgraph

### PR DESCRIPTION
**Pull request by tldavies**

### CODE REVIEW

File: build_children.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#Gets the latest Master build
+GetMasterBuild () {
+
+    BUILD=$(curl -s 'https://api.travis-ci.org/repos/RackHD/'$1'/branches/master' | grep -o '{"branch":{"id":[0-9]*,' | grep -o '[0-9]' | tr -d '\n')
+
+    echo $BUILD
+}
+
+# Get last child project build number
+BUILD_NUM_TASKGRAPH=$(GetMasterBuild on-taskgraph)
```
Comment (benbp) [inline build_children.sh]: `on-http` also uses on-tasks as a dependency.

Comment (tldavies) [inline build_children.sh]: Ok, then I will have on-tasks build it instead of on-core. As on-core builds on-tasks.

**Comment by benbp** [comment]
:+1:

**Comment by tldavies** [comment]
@RackHD/corecommitters @jlongever

**Comment by jlongever** [comment]
+1

**Comment by jlongever** [comment]
Same question as https://github.com/RackHD/on-core/pull/65.  How does the PR commit get applied to the dependent builds?

**Comment by tldavies** [comment]
@jlongever Did my answer in RackHD/on-core#65 answer your question?

**Comment by VulpesArtificem** [comment]
:+1:

**Pull request merged**